pypeit/scripts/show_1dspec.py

```diff
@@ -47,6 +47,8 @@ def main(args):
                     sobjs[ii].RA,
                     sobjs[ii].DEC,
                     sobjs[ii].MASKDEF_OBJNAME)
+            if sobjs[ii].FORCE_EXTRACT is not None and sobjs[ii].FORCE_EXTRACT is True:
```

Comment:
Again, we need to be doing the exact same thing for manual objects, not just deimos manual objects.

Reply:
That's not clear.  There is a difference between the code forced an extraction and the user chose to via the PypeIt file.

Reply:
I added another attribute called `MANUAL_EXTRACT` to SpecObj. This will also show here